src/main/java/org/apache/commons/imaging/formats/xpm/XpmImageParser.java

```diff
@@ -10,7 +10,8 @@
  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
- *  under the License.
+ *
+ * Changed 2015 by Michael Gross, mgmechanics@mgmechanics.de
```

Comment:
On 23.02.2015 19:42, Benedikt Ritter
      wrote:
    
    
      In src/main/java/org/apache/commons/imaging/formats/xpm/XpmImageParser.java:
      > @@ -10,7 +10,8 @@
>   *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
>   *  See the License for the specific language governing permissions and
>   *  limitations under the License.
> - *  under the License.
> + *
> + * Changed 2015 by Michael Gross, mgmechanics@mgmechanics.de

      This has to be removed from all files. Further more (as stated
        in CONTRIBUTING.md) it would be good if you fill out an ICLA
        if you haven't already.
      —
        Reply to this email directly or view
          it on GitHub.
      
        
          
          
        
        
      
    
    
    I'm sorry. I was sure the Apache Software License 2.0 would require
    this. I removed it from all my PRs. The ICLA was filled out,
    printed, underwritten and send to the ASF some weeks ago.
    -- 
Michael Groß